salt/grains/fx2.py

```diff
@@ -30,28 +30,47 @@ def __virtual__():
         return __virtualname__
 
 
+def _find_credentials():
+    '''
+    Cycle through all the possible credentials and return the first one that
+    works
+    '''
+    usernames = []
+    usernames.append(__pillar__['proxy'].get('admin_username', 'root'))
+    if 'fallback_admin_username' in __pillar__.get('proxy'):
+        usernames.append(__pillar__['proxy'].get('fallback_admin_username'))
+
+    for u in usernames:
+        for p in __pillar__['proxy']['passwords']:
+            r = salt.modules.dracr.get_chassis_name(host=__pillar__['proxy']['host'],
```

Comment:
Typically, we prefer variable names that are at least three chars. I don't think we've declared this in the style guide, however, so I will add it.